app/helpers/slug_migrations_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SlugMigrationsHelper
+  def select_options
+    guide_select_options = Guide
+      .with_published_editions
+      .order(:slug).pluck(:slug)
+      .map { |g| [g, g] }
```

Comment:
I'd prefer to leave it for this PR. I've added a placeholder item below the icebox line to remind us to pick it up.